src/chrome/content/rules/administrator.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="administrator.de">
+<target host="*.administrator.de" />
```

Comment:
If website doesn't use any subdomain (except www), most probably they won't in future.